APIMatic.Core/Http/HttpClientWrapper.cs

```diff
@@ -60,6 +60,16 @@ public HttpClientWrapper(ICoreHttpClientConfiguration httpClientConfig)
                 _maximumRetryWaitTime = httpClientConfig.MaximumRetryWaitTime;
                 _client.Timeout = httpClientConfig.Timeout;
             }
+
+            if (httpClientConfig.SkipSslCertVerification)
+            {
+                var httpClientHandler = new HttpClientHandler
+                {
+                    ServerCertificateCustomValidationCallback = (message, cert, chain, errors) => { return true; }
+                };
+
+                _client = new HttpClient(httpClientHandler, disposeHandler: true);
```

Comment:
Why are we overwriting the client's instance after updating the client's timeout and other retries settings on Line 61?

Moreover, think of an approach where you can add the skipSslCertClientHandler into the existing client instead of reseting the provided instance in HttpClientConfiguration.

Reply:
We don't have access to HttpClientHandler from initialized HttpClient object so we cannot update the existing object to skip the SSL certificate verification. This implementation was replicated from the HttpClientWrapper from C# SDK prior to core-lib release.

Reply:
We have moved the HttpClient initialization and configuration to the CoreHttpClientConfiguration Builder.Build method with skipSSLCertificateVerification handling